.travis.yml

```diff
@@ -56,6 +58,11 @@ jobs:
     - env: EMBER_TRY_SCENARIO=ember-canary
     - env: EMBER_TRY_SCENARIO=ember-default-with-jquery
 
+    - stage: "Type tests"
+      env: EMBER_TRY_SCENARIO=typescript-3.6
+    - env: EMBER_TRY_SCENARIO=typescript-3.7
+    - env: EMBER_TRY_SCENARIO=typescript-next
```

Comment:
This doesn't work because we have a separate config file, so it's not invoked correctly. Options:

- pull these back into the main ember-try config file and just specify the `command` argument there
- split them out in the Travis config somehow?

My inclination is the former; it's slightly more verbose but should get the job done just fine and with no tweaking to Travis config other than including the scenarios.

Reply:
The one upside to the latter is that it's possible to run (e.g. locally) `ember try:each --config-path='./config/ember-try-typescript.js` and see that output easily.

Reply:
I worked around this by handling it with some JS (see [below](https://github.com/ember-modifier/ember-modifier/pull/30/files#diff-d20f4baf61c4397c467d6218d0ba6a04R6-R9)).

Reply:
We could also make a single job that does (note: I don't _particularly_ care, just pointing it out as an easy alternative):

```yml
- stage: "Type Tests"
  script: yarn test:types
```

Reply:
Is there any difference in performance? I don’t have a preference myself either, other than wanting to come up with something that’s both easy to maintain and easy to extract for other addons to use as well.

Reply:
I just realized what you were actually saying; there would definitely not be. I prefer this because it means there's one fewer thing to change when changing supported TS versions.

Reply:
On reflection, I changed my mind here because we want to allow one of the environments to fail: `typescript@next`. Related, though: is there a way to have Travis+GitHub notify code owners when an allowed-to-fail build fails? It'd be great if we got notice as early as possible.